app/src/end2endTest/resources/test-mas/claim-378-7101.json

```diff
@@ -0,0 +1,29 @@
+{
+  "dob": "1970-02-19",
+  "firstName": "Janet",
+  "lastName": "Barista",
+  "gender": "female",
+  "collectionId": 378,
+  "veteranIdentifiers": {
+    "icn": "mock1012666073V986378",
+    "ssn": "111-11-1378",
+    "veteranFileId": "9999378",
+    "edipn": "1010234-378",
+    "participantId": "88888378"
+  },
+  "claimDetail": {
+    "benefitClaimId": "1040",
```

Comment:
You need to change either this or the previous 1041. Whatever the intention was.